apps/bfd-pipeline/bfd-pipeline-rif-load/src/test/java/gov/cms/bfd/pipeline/rif/load/RifLoaderIT.java

```diff
@@ -269,6 +269,10 @@ public void loadSampleUUnchanged() {
     DataSource dataSource = DatabaseTestHelper.getTestDatabaseAfterClean();
     loadSample(dataSource, Arrays.asList(StaticRifResourceGroup.SAMPLE_A.getResources()));
     // this should insert a new beneficiary history record
+    /*
+     * FIXME Why is this called "_UNCHANGED" if it will result in a new bene history record? Is the
```

Comment:
@cbrunefearless I believe you landed this, something we should dbl check. Hopefully it's fresher in your mind. Happy to pair and step through it if helpful.